### ISSUE TITLE
Gradient descent, how neural networks learn | Deep learning, chapter 2

### ISSUE THREAD

**Issue by levineuensch** [PXIONG001/Research#20]
### Title

Gradient descent, how neural networks learn | Deep learning, chapter 2

### URL

https://www.youtube.com/watch?v=IHZwWFHWa-w&ab_channel=3Blue1Brown

### Summary 

Basic understanding of how deep learning works, and how we tell the computer what is wrong and how it can fix it.

### Key Points 

Cost function, learning algorithm, calculus methodology
